gpkit/model.py

```diff
@@ -130,80 +126,31 @@ def signomials(self):
                 posynomials.append(constraint)
         return posynomials
 
-    # TODO: replace the below with a dynamically created NomialData 'unsubbed'
-    @property
-    def unsubbed_cs(self):
-        return np.hstack((mag(s.cs) for s in self.signomials))
-
-    @property
-    def unsubbed_exps(self):
-        return functools_reduce(add, (s.exps for s in self.signomials))
-
-    @property
-    def unsubbed_varlocs(self):
-        return locate_vars(self.unsubbed_exps)[0]
-
     @property
-    def unsubbed_varkeys(self):
-        return locate_vars(self.unsubbed_exps)[1]
-
-    @property
-    def variables(self):
-        "All variables currently in the Model."
-        variables = {}
-        for s in self.signomials:
-            variables.update({vk: Variable(**vk.descr) for vk in s.varlocs})
-        return variables
+    def unsubbed(self):
+        return NomialData(signomials=self.signomials, simplify=False)
```

Comment:
The `Model.substitutions` dict and the variable's values have not been applied to them. Open to new names, this convention's been in the code for ages.